src/main/java/uk/gov/hmcts/reform/slc/logging/AppInsights.java

```diff
@@ -102,11 +103,11 @@ public void trackMessageMappedToLetter(String messageId, String serviceName, Str
         );
     }
 
-    public void trackMessageMappedToNull(String messageId, long bodyLength) {
+    public void trackMessageMappedToNull(String messageId) {
```

Comment:
ah derp, had in separate commits but i drooled between branches and this got squashed into one